helm/values.yaml

```diff
@@ -28,19 +42,17 @@ authentication:
     useExternal: false
     externalSecretName: ""
 
-sharedData:
-  ca:
-    secretName: ''
-    path: '/usr/local/share/ca-certificates'
-    key: 'ca.crt'
-  db:
-    host: "localhost"
-    name: "raster"
-    port: 5432
-    sslEnabled: false
-    secrets:
-      useExternal: false
-      externalSecretName: ''
+db:
+  host: "localhost"
```

Comment:
shouldn't we keep those values empty so they can be overriden from sgared-deployments?

Reply:
you are right, Done!